lib/v2/screens/settings/set_currency/set_currency_screen.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'package:seeds/v2/screens/settings/set_currency/interactor/viewmodels/bloc.dart';
+import 'package:seeds/i18n/set_currency.i18n.dart';
+
+class SetCurrencyScreen extends StatefulWidget {
+  const SetCurrencyScreen({Key key}) : super(key: key);
+
+  @override
+  _SetCurrencyScreenState createState() => _SetCurrencyScreenState();
+}
+
+class _SetCurrencyScreenState extends State<SetCurrencyScreen> {
+  SetCurrencyBloc _setCurrencyBloc;
+  final _queryController = TextEditingController();
+
+  @override
+  void initState() {
+    super.initState();
+    _setCurrencyBloc = SetCurrencyBloc();
+    _queryController.addListener(_onQueryChanged);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
+      create: (context) => _setCurrencyBloc,
+      child: Scaffold(
+        appBar: AppBar(
+          title: Text('Select Currency'.i18n),
+          elevation: 0.0,
```

Comment:
is there a way we can declare this in the global theme? I dont want to have to say the appbar elevation is 0.0 each time. 

Do you know?

Reply:
Sounds good

Reply:
nope, lets use your other suggestion 
 appBarTheme: AppBarTheme(elevation: 0.0));